client/src/BuyerTenderList.js

```diff
@@ -57,49 +54,101 @@ const BuyerTenderList = () => {
-								</td>
-								<td>{tender.title}</td>
-								<td>{dateFormat(tender.creation_date)}</td>
-								<td>{dateFormat(tender.announcement_date)}</td>
-								<td>{dateFormat(tender.deadline)}</td>
-								<td>{dateFormat(tender.closing_date)}</td>
-								<td>{tender.description.substring(0, 200)}</td>
-								<td>{tender.no_of_bids_received}</td>
-								<td>£{tender.cost}</td>
-								<td data-status={tender.status}>{tender.status}</td>
-								<td>{dateFormat(tender.last_update)}</td>
-							</tr>
-						))}
-					</tbody>
-				</table>
+				{buyerTenders.length === 0 ? (
+					<div className="msg">No Bid Submitted yet!</div>
+				) : (
+					buyerTenders.map((tender) => (
+						<div className="card" key={tender.tender_id}>
+							<p className="posted-on">
+								Tenser ID: <span className="posted-on-date">{tender.id}</span>
+								<span data-status={tender.status} className="bid-status">
+									{tender.status}
+								</span>
+							</p>
+							<h2 className="title">
+								<a className="tender-id" href={`./bidding/${tender.id}`}>
+									{tender.title}
+								</a>
+							</h2>
+							<div className="details">
+								<p>
+									<strong>Tender Creation Date:</strong>{" "}
+									{new Date(tender.creation_date).toLocaleDateString()}
+								</p>
+								<p>
+									<strong>Tender Announcement Date: </strong>
+									{new Date(tender.announcement_date).toLocaleDateString()}
+								</p>
+								<p>
+									<strong>Tender Deadline Date: </strong>
+									{new Date(tender.deadline).toLocaleDateString()}
+								</p>
+								<p>
+									<strong>Tender Closing Date: </strong>
+									{new Date(tender.closing_date).toLocaleDateString()}
+								</p>
+							</div>
+							<h4>Description:</h4>
+							<p className="cover-letter">
+								{" "}
+								{expandedTenderId === tender.id ? (
+									<p>
+										{tender.description || "No description available"}
+										<button
+											className="toggle-text"
+											onClick={() => handleTenderClick(tender.id)}
+											aria-expanded={expandedTenderId === tender.id}
+											aria-controls={`description-${tender.id}`}
+										>
+											Read less
+										</button>
+									</p>
+								) : (
+									<p>
+										{truncateText(
+											tender.description || "No description available",
+											150
+										)}
+										{(tender.description || "").length > 30 && (
+											<button
+												className="toggle-text"
+												onClick={() => handleTenderClick(tender.id)}
+												aria-expanded={expandedTenderId === tender.id}
+												aria-controls={`description-${tender.id}`}
+											>
+												Read More
+											</button>
+										)}
+									</p>
+								)}
+							</p>
+							<p>
+								<strong>Cost: </strong>£{tender.cost}{" "}
+								<span>
+									<strong>No. Of Bids Received: </strong>
+									{tender.no_of_bids_received}
+								</span>{" "}
```

Comment:
Why is there a {" "} here?

Reply:
Who adds them? Your editor? You might want to check if there is a configuration you can tune so they are not added, since they are unnecessary.